plone-5/instance/src/ukstats.sparql_dataconnector/README.rst

```diff
@@ -0,0 +1,94 @@
+.. This README is meant for consumption by humans and pypi. Pypi can render rst files so please do not use Sphinx features.
```

Comment:
README is a bit bogus

Reply:
Updated the README to reflect actual extension content